astropy/cosmology/funcs.py

```diff
@@ -113,8 +113,8 @@ def z_at_value(func, fval, zmin=1e-8, zmax=1000, ztol=1e-8, maxfun=500,
     zmax. We use log spacing to adequately sample the steep part of
     the curve at low distance moduli:
 
-    >>> zmin = z_at_value(Planck13.distmod, Dvals.min())
-    >>> zmax = z_at_value(Planck13.distmod, Dvals.max())
+    >>> zmin = scalar_z_at_value(Planck13.distmod, Dvals.min())
```

Comment:
Are you sure you want to make these changes? Maybe instead just move the docstring to `z_at_value`? Now `z_at_value` will get the wrong examples!

Reply:
I'm definitely not sure. I think both functions might merit somewhat different docstrings. I also think that the more verbose docstring, with examples, should be moved to `z_at_value`. I was also thinking about using `format_doc` to fix up the examples.

Reply:
Done!